.github/workflows/tests.yml

```
@@ -339,14 +377,24 @@ jobs:
           conda create -n blarg1 -yq python=2.7
           conda create -n blarg3 -yq python=3.7
           conda create -n blarg4 -yq python nomkl numpy pandas svn
-          pytest --color=yes -v -n auto --basetemp /tmp/cb --cov conda_build --cov-append --cov-report xml -m "not serial" tests "${PYTEST_REPLAY_OPTIONS[@]+"${PYTEST_REPLAY_OPTIONS[@]}"}"
+          pytest --color=yes -v -n auto --basetemp /tmp/cb --cov conda_build /
```

replace all / at the end of lines with \ to continue to the next line

```suggestion
          pytest --color=yes -v -n auto --basetemp /tmp/cb --cov conda_build \
```